cloud/configs/nginx.conf

```diff
@@ -14,9 +14,13 @@ server {
     access_log /var/log/nginx/etagecom.io_access.log;
     error_log /var/log/nginx/etagecom.io_error.log;
 
+    #CANNOT redirect POST messages (they change to GETs). Reject instead.
+    if ($http_x_forwarded_proto != "https" & $request_method = POST) {
+       return 405 Allow "GET" always;
```

Comment:
This line also causes an error:

`invalid number of arguments in "return" directive in /etc/nginx/sites-enabled/nginx.conf:19`